cmd/ci-scheduling-webhook/mutation.go

```diff
@@ -203,74 +203,24 @@ func mutatePod(w http.ResponseWriter, r *http.Request) {
 				if requiresNetAdmin {
 					// Build the correct patch based on existing securityContext
 					if container.SecurityContext == nil {
-						// No securityContext exists, create one with capabilities and runAsUser
-						securityContext := map[string]interface{}{
-							"runAsUser":    int64(0),
-							"runAsNonRoot": false,
-							"capabilities": map[string]interface{}{
-								"add": []string{"NET_ADMIN", "NET_RAW"},
+						// No securityContext exists, create one with capabilities
+						container.SecurityContext = &corev1.SecurityContext{
```

Comment:
i would just skip if the SCC does not exist... creating SCC requires certain privileges and I'd rather create them manually/via apply-config

Reply:
Justin's answer:
It is not creating an SCC, it is configuring a pod's security context.
The necessary SCC to support the request is applied via gitops